Allow the setup script to replace the interpreter path

Fixes Arch Linux package where the “python” command is Python 3 by default
and Python 2 is run less ambiguously with “python2”. See
http://docs.python.org/distutils/setupscript.html#installing-scripts.

=== modified file 'setup.py'
--- setup.py	2010-02-25 12:20:40 +0000
+++ setup.py	2011-09-15 12:26:06 +0000
@@ -4,8 +4,8 @@
 	name='iview',
 	version=iview.config.version,
 	packages=['iview'],
+	scripts=['iview-cli', 'iview-gtk'],
 	data_files=[
-			('/usr/bin', ['iview-cli', 'iview-gtk']),
 			('/usr/share/applications', ['iview-gtk.desktop']),
 		],
 	)

